### PR TITLE
fix: custom configuration of crio in testing cluster

### PR DESCRIPTION
**Pull request by ksimon1**
**What this PR does / why we need it**:
our e2e are not working due to permission issue in crio.
https://github.com/cri-o/cri-o/pull/7192 should revert the change.
Until that revert PR is released and available in ci envs, we have to
set special configuration on crio on all nodes
**Release note**:
```
NONE

```

### CODE REVIEW

File: automation/test.sh [hunk 1]
```diff
@@ -70,11 +70,12 @@ function latest_version() {
 export KUBEVIRT_VERSION=$(latest_version "kubevirt")
 
 # Latest released CDI version
-export CDI_VERSION=$(latest_version "containerized-data-importer")
+#export CDI_VERSION=$(latest_version "containerized-data-importer")
```
Comment (jcanocan) [inline automation/test.sh:73]: If it's no longer required, we should drop this line to keep the file clean.

Comment (ksimon1) [reply automation/test.sh:73]: This is just testing PR, which should investigate where is the problem in CI. This change will not be part of result PR

**Comment by ksimon1** [comment]
This PR is no longer needed

**Pull request closed without merging**